pkgs/development/python-modules/ifcfg/default.nix

```diff
@@ -0,0 +1,32 @@
+{ lib
+, fetchFromGitHub
+, python3Packages
```

Comment:
This was correct before; the dependencies should be listed as separate arguments. Sorry, I should have mentioned this when commenting on @azahi's patch.